e2e/e2e_test.go

```diff
@@ -983,32 +1016,33 @@ func Test_Hedera_Native_EVM_NFT_Transfer(t *testing.T) {
 		},
 	}
 	// and recreate signed auth message
-	authMsgBytes, err := auth_message.
-		EncodeNftBytesFrom(
-			expectedTxRecord.SourceChainID,
-			expectedTxRecord.TargetChainID,
-			expectedTxRecord.TransactionID,
-			expectedTxRecord.TargetAsset,
-			expectedTxRecord.SerialNumber,
-			expectedTxRecord.Metadata,
-			expectedTxRecord.Receiver)
+	authMsgBytes, err := auth_message.EncodeNftBytesFrom(
+		expectedTxRecord.SourceChainID,
+		expectedTxRecord.TargetChainID,
+		expectedTxRecord.TransactionID,
+		expectedTxRecord.TargetAsset,
+		expectedTxRecord.SerialNumber,
+		expectedTxRecord.Metadata,
+		expectedTxRecord.Receiver,
+	)
 	if err != nil {
 		t.Fatalf("[%s] - Failed to encode the authorisation signature. Error: [%s]", expectedTxRecord.TransactionID, err)
 	}
 
-	// Wait a bit
-	time.Sleep(20 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.